src/MarkdownTextInput.tsx

```diff
@@ -28,10 +28,9 @@ function initializeLiveMarkdownIfNeeded() {
   if (initialized) {
     return;
   }
-  if (!NativeLiveMarkdownModule) {
-    throw new Error('[react-native-live-markdown] NativeLiveMarkdownModule is not available');
+  if (NativeLiveMarkdownModule) {
+    NativeLiveMarkdownModule.install();
```

Comment:
What's the reason for this change? If `NativeLiveMarkdownModule` is not injected properly, we'd like to fail immediately instead of showing a non-actionable error "global.jsi_setMarkdownRuntime is not available" that sounds more like an actual implementation bug rather than setup issue.

Reply:
This is required for the mock to work. Otherwise we'd have to [mock react-native TurboModuleRegistry](https://github.com/Expensify/App/blob/4a9bd7cb3bbbbd95ec7b9a229611786ebe4dbbe8/__mocks__/react-native.ts#L45-L59).